### PR TITLE
CLDC-1809 Add prevshared question

### PR DESCRIPTION
**Pull request by ghost**
this needs to be added to the new year form rather than the current ones https://digital.dclg.gov.uk/jira/browse/CLDC-1809

Ask this question if user owned previous property

### CODE REVIEW

File: app/models/form/sales/questions/prevshared.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Form::Sales::Questions::Prevshared < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "prevshared"
+    @check_answer_label = "Buyers previously owned a shared property"
```
Comment (natdeanlewissoftwire) [inline app/models/form/sales/questions/prevshared.rb]: should this be Buyer/Buyers depending if it's a joint purchase? or does this only occcur for shared ownerships already

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm, just one query

File: app/models/form/sales/questions/prevshared.rb (hunk 1 above)
Comment (ghost) [reply app/models/form/sales/questions/prevshared.rb]: good point, I will figure that out!

File: app/models/form/sales/subsections/income_benefits_and_savings.rb [hunk 2]
```diff
@@ -22,6 +22,15 @@ def pages
...
       Form::Sales::Pages::PreviousOwnership.new(nil, nil, self),
-    ]
+      previous_shared_page,
+    ].compact
+  end
+
+private
+
+  def previous_shared_page
+    if form.start_date.year >= 2023
```
Comment (natdeanlewissoftwire) [inline app/models/form/sales/subsections/income_benefits_and_savings.rb:32]: should this be the `collection_start_year` rather than `start_date.year`?

Comment (natdeanlewissoftwire) [reply app/models/form/sales/subsections/income_benefits_and_savings.rb:32]: ah wait no you're right I was thinking of `log.startdate`

**Pull request merged**